include/Graphs/CFLGraph.h

```diff
@@ -78,10 +78,10 @@ class CFLGraph: public GenericCFLGraphTy
 public:
     typedef CFLGrammar::Symbol Symbol;
     typedef GenericNode<CFLNode,CFLEdge>::GEdgeSetTy CFLEdgeSet;
-    Map<std::string, Symbol> label2SymMap;
-    Map<Symbol, std::string> sym2LabelMap;
-    Symbol startSymbol;
-    Map<CFLGrammar::Kind,  Set<CFLGrammar::Attribute>> kind2AttrMap;
+    Map<std::string, Symbol> label2KindMap;
```

Comment:
why do we need `label2KindMap ` and `kind2LabelMap ` maps in `CFLGraph`? They are supposed to be in `CFLGraphBuilder` only?